src/screens/Home.tsx

```
@@ -81,23 +79,43 @@ export default function Home() {
+    loadingMore,
+    hasNextPage,
+    loadMore,
+    refresh,
+    refreshing
+  } = useInfiniteScroll({
+    fetchData: fetchPostsData,
+    pageSize: 10
+  });
 
+  // Refresh data when timeline type changes
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: refresh

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)